cnho_stables/chain.json

```diff
@@ -0,0 +1,112 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "cnho stables",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://cnho.io/",
+  "pretty_name": "cnho stables",
```

Comment:
No capital letters? Not supposed to be something like "CNHO Stables"?

Reply:
use CNHO Stables is fine?